src/gwt/src/org/rstudio/studio/client/workbench/views/files/model/FilesServerOperations.java

```diff
@@ -98,4 +98,9 @@ void writeConfigJSON(String relativePath,
    void readConfigJSON(String relativePath,
                  boolean logErrorIfNotFound,
                  ServerRequestCallback<JavaScriptObject> requestCallback);
+
+   // Use VERY sparingly; we generally don't want to expose
+   // non-aliased paths to other parts of the client codebase
+   // (most client-side APIs assume paths are aliased)
+   String resolveAliasedPath(FileSystemItem file);
```

Comment:
Nit: If you give this a doxygen-style comment (i.e. `/**`) it will show up in IDEs to help discourage use during autocomplete.

Reply:
Done!